hedera-node/data/config/node.properties

```diff
@@ -3,3 +3,4 @@ hedera.recordStream.logDir=data/recordstreams/
 hedera.recordStream.sidecarDir=
 hedera.recordStream.queueCapacity=5000
 netty.mode=TEST
+blockStream.blockFileDir=data/blockstreams/
```

Comment:
most places have "blockStreams", this one says "blockstreams". Is this intentional?

Reply:
AFAIK `data/recordstreams` is intentionally set this way, but it seemed like an error to me. I'll change both

Reply:
I substituted a capital 'S' for both properties

Reply:
I think it should be `data/blockStreams` like Richard pointed

Reply:
Done